cmd/otelcol/Dockerfile.windows

```diff
@@ -0,0 +1,46 @@
+FROM mcr.microsoft.com/windows/servercore:1809
+
+# Setting PowerShell as a default executor. 
+SHELL ["powershell", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+WORKDIR "C:\\Program Files\Splunk\OpenTelemetry Collector"
+
+# Copy the pre-built local binary
+COPY otelcol.exe ./
+
+# Copy the local config
+COPY config/collector/gateway_config.yaml ./cmd/otelcol/config/collector/gateway_config.yaml
+COPY config/collector/otlp_config_linux.yaml ./cmd/otelcol/config/collector/otlp_config_linux.yaml
+COPY config/collector/agent_config.yaml ./cmd/otelcol/config/collector/agent_config.yaml
+COPY config/collector/fargate_config.yaml ./cmd/otelcol/config/collector/fargate_config.yaml
+
+WORKDIR "C:\ProgramData\Splunk\OpenTelemetry Collector"
+COPY config/collector/gateway_config.yaml ./
+COPY config/collector/otlp_config_linux.yaml ./
+COPY config/collector/agent_config.yaml ./
+COPY config/collector/fargate_config.yaml ./
```

Comment:
We probably won't need the configs in `C:\\Program Files\Splunk\OpenTelemetry Collector`, so just copy the configs directly to `C:\ProgramData\Splunk\OpenTelemetry Collector`.

Reply:
Ok removed it.